docs/user-docs/latest/plugins/strapi-plugins.md

```diff
@@ -51,5 +56,85 @@ The Users & Permissions plugin impacts several parts of the admin panel. The tab
+
+## Additional plugins
+
+### <img width="28" src="../assets/plugins/Documentation-swagger.png"> Documentation
+
+ The Documentation plugin automates documentation for APIs in a Strapi application using the Open API specification version 3.0.1. When the Documentation plugin is installed it is available in the admin panel, under the heading plugins. The Documentation plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-documentation).  The Documentation plugin allows you to:
+
+- open the API documentation,
+- regenerate the documentation,
+- restrict access to the documentation endpoint.
+
+The Documentation plugin affects multiple parts of the admin panel. The table below lists all the additional options and settings that are added to a Strapi application once the plugin has been installed.
+
+| Section impacted    | Options and settings         |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| Documentation    | <ul>Addition of a new Documentation option in the main navigation under the plugins heading, which contains links to open and refresh the documentation.   </ul>        |
+| Settings     | <ul><li>Addition of a "Documentation plugin" setting section, which controls whether the documentation endpoint is private or not. <br> 👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > documentation plugin* </li><br>  <li> Activation of role based access control for accessing, updating, deleting, and regenerating the documentation. Administrators can authorize different access levels to different types of users in the *Plugins* tab and the *Settings* tab. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Administration Panel > Roles*  <BronzeBadge link="https://strapi.io/pricing-self-hosted"/> <SilverBadge link="https://strapi.io/pricing-self-hosted"/> <GoldBadge link="https://strapi.io/pricing-self-hosted"/> </li></ul>|
+
+### <img width="28" src="../assets/plugins/Gatsby_Monogram.png"> Gatsby preview
+
+The Gatsby preview plugin allows applications with Gatsby Cloud accounts to preview the front end. The Strapi Gatsby preview plugin is available in the in-app Marketplace and the [Strapi Market](https://market.strapi.io/plugins/@strapi-plugin-gatsby-preview).
+
+| Section impacted    | Options and settings         |
+|------------|-----------------|
+| Settings     |  Addition of the Gatsby preview plugin section in the Settings sub navigation. In the plugin settings the user can: <ul><li> enable or disable Collection Types and Single Types, </li> <li>add the Gatsby Content Sync URL</li> </ul> |
```

Comment:
```suggestion
| Settings     |  Addition of a "Gatsby preview plugin" setting section, enables/disables collection types and single types, and allows to add the Gatsby Content Sync URL. <br>👉 Path reminder: ![Settings icon](../assets/icons/settings.svg) *Settings > Gatsby preview plugin* |
```

Can you check that the setting section name is "Gatsby preview plugin" written like so? :)